src/status_im2/common/standard_authentication/enter_password/view.cljs

```diff
@@ -9,7 +9,7 @@
     [utils.re-frame :as rf]))
 
 (defn view
-  [{:keys [on-enter-password button-label button-icon-left customization-color]}]
+  [{:keys [on-enter-password button-label button-icon-left profile-color customization-color]}]
```

Comment:
why do we need profile color too? surely we can just pass this as customization or what's the difference here? 🤔

Reply:
I need to remove this and use the color from the existing sub `:profile/multiaccount` in the component.